Scripts/package.py

```diff
@@ -106,9 +106,18 @@ def download(self):
             print("cached!")
         else:
             flush("downloading {:.2f}MB...".format(self.MB), end=' ')
-            with request.urlopen(self.url) as reader:
-                with open(self.z3_zip, mode="wb") as writer:
-                    writer.write(reader.read())
+            maxTries = 5
```

Comment:
At this point it might make sense to put this into a global constant. (With L132)